src/data/it/windows_versions.rs

```diff
@@ -5,6 +5,26 @@ use wasm_bindgen::prelude::*;
 pub fn windows_version() -> String {
 	WINDOWS_VERSIONS[seeder::gen_range(0..WINDOWS_VERSIONS_LEN)].to_string()
 }
+
+#[wasm_bindgen]
+pub fn client_version() -> String {
+	CLIENT_VERSIONS[seeder::gen_range(0..CLIENT_VERSIONS_LEN)].to_string()
+}
+
+#[wasm_bindgen]
+pub fn server_version() -> String {
+	SERVER_VERSIONS[seeder::gen_range(0..SERVER_VERSIONS_LEN)].to_string()
+}
+
+#[wasm_bindgen]
+pub fn specialized_version() -> String {
+	SPECIALIZED_VERSIONS[seeder::gen_range(0..SPECIALIZED_VERSIONS_LEN)].to_string()
+}
+
+#[wasm_bindgen]
```

Comment:
Let's prefix the js name with windows ie `#[wasm_bindgen(js_name = windows_client_version)]`. These function names none prefixed feel as though they'll be common for other major systems like linux as well.